### PR TITLE
Migrate in-meeting app to V3

### PR DESCRIPTION
**Pull request by JerryYangKai**

### CODE REVIEW

File: hello-world-in-meeting/teamsfx/settings.json [hunk 1]
```diff
@@ -0,0 +1,4 @@
+{
+    "version": "3.0.0",
+    "trackingId": "bd4dafa3-0373-402a-b88d-ada81fc99a56"
```
Comment (yukun-dong) [inline hello-world-in-meeting/teamsfx/settings.json:3]: This tracking Id should be removed. It will be added by Coordinator when downloading sample.

Comment (hund030) [inline hello-world-in-meeting/teamsfx/settings.json:3]: Remove tracking Id.

File: hello-world-in-meeting/teamsfx/app.yml [hunk 2]
```diff
@@ -0,0 +1,69 @@
...
+      args: run build --if-present
+  - uses: azureStorage/deploy # Deploy bits to Azure Storage Static Website
+    with:
+      distributionPath: ./build # Deploy base folder
+      resourceId: ${{TAB_AZURE_STORAGE_RESOURCE_ID}} # The resource id of the cloud resource to be deployed to
+
+registerApp:
+  - uses: teamsApp/create # Creates a Teams app
+    with:
+      name: tab
```
Comment (hund030) [inline hello-world-in-meeting/teamsfx/app.yml:37]: Same here.

File: hello-world-in-meeting/teamsfx/app.local.yml [hunk 3]
```diff
@@ -0,0 +1,42 @@
+version: 1.0.0
+
+registerApp:
+  - uses: teamsApp/create # Creates a Teams app
+    with:
+      name: tab # Teams app name
```
Comment (hund030) [inline hello-world-in-meeting/teamsfx/app.local.yml:6]: Is the name expected?

**Review by yukun-dong — APPROVED** [review]

**Pull request merged**